src/eom-image.c

```diff
@@ -61,9 +61,6 @@
 
 #if defined(HAVE_LCMS) && defined(GDK_WINDOWING_X11)
 #include <lcms2.h>
-#ifndef EXIF_TAG_GAMMA
-#define EXIF_TAG_GAMMA 0xa500
-#endif
```

Comment:
EXIF_TAG_GAMMA was added on libexif-0.6.22, eom requires libexif-0.6.14 ([REF](https://github.com/libexif/libexif/blob/303643aebb75b2a9f3db83b774b1d448fdfe694b/NEWS#L12))
```
$ git grep LIBEXIF_REQUIRED=
configure.ac:LIBEXIF_REQUIRED=0.6.14
```
This macro is used in another file, but it doesn't check whether it's defined:
```
$ git grep -n EXIF_TAG_GAMMA
src/eom-metadata-reader-jpg.c:608:                      entry = exif_data_get_entry (exif, EXIF_TAG_GAMMA);
```

Reply:
Libexif required version is updated.